flake.nix

```diff
@@ -0,0 +1,128 @@
+{
+  description = "Medusa smart-contract fuzzer";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixos-22.11";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; config.allowUnfree = true; };
+        pyCommon = {
+          format = "pyproject";
+          nativeBuildInputs = with pkgs.python39Packages; [ pythonRelaxDepsHook ];
+          pythonRelaxDeps = true;
+          doCheck = false;
+        };
+      in
+      rec {
+
+        packages = rec {
+
+          solc-select = pkgs.python39Packages.buildPythonPackage (pyCommon // {
+            pname = "solc-select";
+            version = "1.0.3";
+            src = builtins.fetchGit {
+              url = "git+ssh://git@github.com/crytic/solc-select";
+              rev = "97f160611c39d46e27d6f44a5a61344e6218d584";
+            };
+            propagatedBuildInputs = with pkgs.python39Packages; [
+              packaging
+              setuptools
+              pycryptodome
+            ];
+          });
```

Comment:
In-line addition of nix support to `solc-select` & other packages that don't feature a `flake.nix` file yet. If/when they do, these pieces of config can move into their respective repos & can be removed from this project.
In the meantime, they allow `medusa` to depend on a specific commit of `crytic-compile`, etc